src/main/java/com/example/flipcardsapp/FlipCard/FlipCardBack.java

```diff
@@ -0,0 +1,37 @@
+package com.example.flipcardsapp.FlipCard;
+
+import javax.persistence.*;
+
+@Entity
+public class FlipCardBack implements FlipCard {
```

Comment:
why are you implementing empty interface to entity class??

Reply:
I thought It would be needed, but it's not. Remove

Reply:
we are implementing interfaces to logic, in entity we can extend class